clang/lib/CodeGen/CGExpr.cpp

```diff
@@ -3292,22 +3307,39 @@ LValue CodeGenFunction::EmitPredefinedLValue(const PredefinedExpr *E) {
 ///   { i16 TypeKind, i16 TypeInfo }
 /// \endcode
 ///
-/// followed by an array of i8 containing the type name. TypeKind is 0 for an
-/// integer, 1 for a floating point value, and -1 for anything else.
+/// followed by an array of i8 containing the type name with extra information
+/// for BitInt. TypeKind is 0 for an integer, 1 for a floating point value, 2
+/// for BitInt and -1 for anything else.
```

Comment:
Do you need to mention the literal numeric values since they're enum constants now?

Reply:
I think it is important that those values are 16 bit integers. I'll replace with TK_BitInt(2).

Reply:
It was replaced and i hope it will make comment clear.